<launch>
    <env name="ROSCONSOLE_CONFIG_FILE" value="$(find nav_planner)/logging_debug.conf"/>

    <node pkg="octomap_server" type="octomap_server_node" name="octomap_server" output="screen">
		<param name="resolution" value="0.05" />
		<param name="frame_id" type="string" value="/odom" />
		<param name="sensor_model/max_range" value="10.0" />
		<param name="latch" value="true" />

		<remap from="cloud_in" to="/camera/depth/points" />
	</node>
    
    <node pkg="nav_planner" type="goal_identifier_node" name="goal_identifier" output="screen">
		<remap from="octomap" to="/octomap_full" />
        <remap from="position" to="/tf" />
	</node>

	<node pkg="nav_planner" type="velocity_control_node" name="velocity_Controller" output="screen">
		<remap from="keyop/motor_power" to="mobile_base/commands/motor_power"/>
		<remap from="keyop/cmd_vel" to="mobile_base/commands/velocity"/>
		<remap from="position" to="/tf" />

		<param name="linear_vel_max"   value="1"  type="double"/>
		<param name="angular_vel_max"  value="1.57"  type="double"/>
		<param name="velocity_constant"  value="2"  type="double"/>
		<param name="angularv_constant"  value="2"  type="double"/>
  	</node>
</launch>

